### PR TITLE
Add delivery employee service and get/create/update/delete endpoints

### PR DESCRIPTION
**Pull request by LMBishop**
Solves 1, 2, 3, 4, and 5

### CODE REVIEW

File: src/main/java/org/kainos/ea/api/DeliveryEmployeeService.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.kainos.ea.client.InvalidDeliveryEmployeeException;
+import org.kainos.ea.core.DeliveryEmployeeValidator;
+import org.kainos.ea.db.DeliveryEmployeeDao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class DeliveryEmployeeService {
+
+    private final DeliveryEmployeeDao deliveryEmployeeDao = new DeliveryEmployeeDao();
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/api/DeliveryEmployeeService.java]: Is there a way you can improve this to follow SOLID principles. For example you could use dependency injection here

File: src/main/java/org/kainos/ea/client/FailedToCreateDeliveryEmployeeException.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.kainos.ea.client;
+
+public class FailedToCreateDeliveryEmployeeException extends Exception {
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/client/FailedToCreateDeliveryEmployeeException.java:3]: Missing a getMessage() override

File: src/main/java/org/kainos/ea/client/FailedToGetDeliveryEmployeesException.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package org.kainos.ea.client;
+
+public class FailedToGetDeliveryEmployeesException extends Exception {
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/client/FailedToGetDeliveryEmployeesException.java:3]: Missing a getMessage() override

File: src/main/java/org/kainos/ea/client/DeliveryEmployeeDoesNotExistException.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package org.kainos.ea.client;
+
+public class DeliveryEmployeeDoesNotExistException extends Exception {
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/client/DeliveryEmployeeDoesNotExistException.java:3]: Missing a getMessage() override

**Comment by LMBishop** [comment]
Also now solves 2

**Comment by LMBishop** [comment]
and 5

**Review by shaunganley — APPROVED** [review]

**Review by Adaray786 — APPROVED** [review]

**Pull request merged**